src/styles/global/ie11-fixes.styl

```diff
@@ -0,0 +1,6 @@
+html
+body
+#app-container
+#app-container > div
+  height 100%      //Enables display: flex child items to grow properly in IE11.
+  min-height 100%  //Prevents the above fix from limiting the height of said elements in other browsers.
```

Comment:
@shaunanoordin sorry, poor communication on my part. I'll push the latest I've got to add-bootstrap PR, so you can have a look. Btw bootstrap should already take care of what you're describing.